docker-compose.yml

```diff
@@ -4,7 +4,7 @@ services:
     image: korerorero-front-end:latest
     container_name: korerorero-front-end
     build:
-      context: https://github.com/ServiceInnovationLab/korerorero-front-end.git#install-chat-ui
+      context: https://github.com/ServiceInnovationLab/korerorero-front-end.git#v0.1.0
```

Comment:
```suggestion
      context: https://github.com/ServiceInnovationLab/korerorero-front-end.git
```

Reply:
![image](https://user-images.githubusercontent.com/13788498/79818995-30535480-83dd-11ea-8019-0ee050e97f51.png)

Reply:
I forgot to push the `v0.1.0` tag. Try it again with the tag?

Reply:
:+1: